FLOE-493: first draft of making/hacking article.

waharnum opened the pull request:
Reference JIRA: https://issues.fluidproject.org/browse/FLOE-493

simonbates commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,140 @@
+
+### Hackathon
+
+We use the term "hackathon" expansively in this entry to refer to events that emphasize intensive short-term collaboration between small groups, usually (but not always) physically co-located. The groups typically ideate, design and prototype around themes such as <a href="http://arthackathon.co.uk/" target="_blank" class="link-external">art</a> or <a href="http://civictech.ca/2016/04/08/codeacro ...
```
"then present their result in some form to to the larger gathering throughout the event or at the end"

Double "to".

michelled replied:
the double 'to' is still here

simonbates commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,140 @@
+While these events are often focused on producing or proposing new software or hardware as a means of addressing a social problem or creating a new product, this is not stricly necessary; hackathons have been held to produce
+<a href="https://www.policyhack.com.au/" target="_blank" class="link-external">government policy</a>, or ideas to address <a href="http://foodinnovationprogram.org/hacking-hunger/" target="_blank" class="link-external">food security</a>.
+
+In learning settings hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution.
```
"course curricula" maybe?

simonbates commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,140 @@
+category: Methods
+---
+
+Hackathons, makeathons and similar events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world.
```
maybe: "and in the broader world"

waharnum commented:
Attaching my notes on the design crit on this entry
---

# Notes from the Design Crit

## Guests

* Liz Delarosa from Queen's University joined us

## Terminology

"hackathon" and "hacking" may themselves be alienating terms and have associations with the need for technical knowledge to participate. We discussed alternatives (such as "collabathon"), but didn't come to a definitive conclusion.

For now, we will probably stick with "hackathon" and "hacking", but add further content regarding:
  * How we are using the terms and why
  * Why the terms themselves need some critical examination

## Remote Collaboration for Hacking & Making

The current language emphasizes "physical proximity" as a characteristic of hackathons. This isn't particularly inclusive and it would be best to:
  * Remove this language
  * Add some content on specific tools and approaches for remote participation

It was suggested that events and initiative that are aligned with hacking/making's spirit without being called "hackathons" (like Global Game Jam) be mentioned as themed events with distributed teams and groups.

## Participant Motivations

The differing motivations of participants in these activities were discussed - this is especially so when hacking/making targets the "needs" of individuals for educational purposes (like engineering/OT students working with older adults, in Liz Delarosa's example).

The question: how to design to account for different motivations for participating, and make it clear to participants what they're actually signing up for? What kind of information do they need to be given?

## "Nothing About Us Without Us"

It was suggested that some additional language be added around this concept  acknowledging the extra work involved, but also that it will lead to better outcomes and learning.

Another point along these lines that came up: the entry should discuss how designing for the needs of extreme users creates benefits for all (curb cut effect), and innovations that wouldn't occur otherwise (innovation from the margins)

## De-centering Hackathons from the Entry

The entry as-is skews towards hackathons in terms of how it presents hacking and making. It's important to keep in mind the broader umbrella of:

### Hacking, Making and DIY Learning Generally

We're trying to talk about bringing inclusive design to a class of activities and events that have some or all of these characteristics:

* DIY ideology: self-motivated, not mass produced, not group directed
* pressurized creativity
* one to one design (both as an outcome, and to allow more diversity in participation)
* Informal creation done to make something for someone who needs it (such as Tetra Society)
* "an engagement and relationship with constructing new things" (Simon)

### Hackathons are Not a Substitute for Other Things

Things missing from hackathons:

* Time
* Depth
* Field Work

Questions that arise from this:
* What do we need in addition to hackathons?
* What do we do *after* the hackathon?

Overall, hackathons should be a class of event discussed here; we need other examples and need the entry to focus upon the *kind* of learning activities (making, hacking, experiential learning, Papert-style constructionism, etc) moreso than a particular event.

## Splitting Into Multiple Entries

Not at this stage, but may need to happen at some point. It was pointed out there's a lot of common things about running inclusive events that might best have their own entry (captioning, physical space configuration, etc).

## Scenarios Need Work (or Perhaps Deletion)

* If we want to retain these, it would be best to get them reviewed by practitioners.
* They are mostly specific to disability and accessibility
* We need to be careful with the specialist terminology (like "multimodal interfaces"); define these terms inline, or link them out to definitions in the ILDH or otherwise.
* Generally speaking, examples are better than scenarios.

## Having More Tools Without Becoming a Clearinghouse

The intention of the tools section (and the distinction from Resources) needs some clarity. It would also be good to add some additional tools, and explain in more detail some things about the tools such as:

* Who might use them ("this tool would likely be used by developers building software")
* General category the tools fall into (software, fabrication, design activities...)

It would be good to have some tools that are more oriented towards beginners / non-technical people like:

* Scratch
* Makey Makey

Some OT/AT/3d printing/physical hacking approaches might also be interesting:

* Thingiverse
* Cardboard for making / making on the cheap
* Ikea hacking
* Go Baby Go: http://sites.udel.edu/gobabygo/

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,172 @@
+
+#### Inclusive Design Guide
+
+The <a href="https://guide.inclusivedesign.ca/" target="_blank" class="link-external">Inclusive Design Guide</a> is an evolving collection of insights, practices, tools and activities that can be applied a wide range of design challenges in the digital, physical and service realm.
```
I think it might make sense to flesh this section out a bit, perhaps with the help of @dayotte. In particular, I'm thinking that we could describe how the Inclusive Design Guide could be used within the context of maker events and workshops to help catalyze the inclusive design process—as a tool for generating or evaluating ideas.

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+## Inclusively Designing Hackathons and Hacking
+
+It's been said about hackathons that the connections and ideas that get made between people attending are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
```
I often feel a bit weird about using the phrase "it's been said..." rather than a more specific citation. Maybe in this case, it would be better to just go ahead and own the (quite interesting and reasonable) assertion that the important outcome of a hackathon is the connections, ideas, starting points and relationships that are formed from the process?

waharnum replied:
Agreed - "it has been said" is a somewhat evasive phrase to make a claim without either asserting it personally or providing citations. Will revise.

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+category: Methods
+---
+
+Hackathons, makeathons and similar events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world.
```
What makes a "hackathon" and a "makeathon" similar? Is there an opportunity here to summarize just in a word or two what these kinds of events are about? Are they "community-oriented" events? Or "participatory"? What's common to them all?

colinbdclark replied:
And why are these kinds of events useful for education? What kind of learning opportunities do they afford that perhaps traditional classroom teaching doesn't? And do the present an opportunity to be somehow more inclusive, personalized, or open? (again, one sentence is plenty, but I'm thinking this little introduction should try to distill a few of the key concepts here very briefly)

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+### Hackathon
+
+We use the term "hackathon" expansively in this entry to refer to events that emphasize intensive short-term collaboration between small groups. The groups typically ideate, design and prototype around themes such as <a href="http://arthackathon.co.uk/" target="_blank" class="link-external">art</a> or <a href="http://civictech.ca/2016/04/08/codeacross-toronto-community-collaboration-in-celebratio ...
```
Is "ideate" too much of a buzzword? Would "brainstorm" be more recognizeable to non-designers?

colinbdclark replied:
Perhaps: "...around social, cultural, or political issues that are relevant to them (for example, art and open data)..."

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+While these events are often focused on producing or proposing new software or hardware as a means of addressing a social problem or creating a new product, this is not stricly necessary; hackathons have been held to produce
+<a href="https://www.policyhack.com.au/" target="_blank" class="link-external">government policy</a>, or ideas to address <a href="http://foodinnovationprogram.org/hacking-hunger/" target="_blank" class="link-external">food security</a>.
+
+In learning settings hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution.
```
Missing comma between "In learning settings" and "hackathons" in the first sentence.

colinbdclark replied:
What's the educational benefit to holding hackathons, do you think?

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+### Hacking
+
+We use the term "hacking" expansively here to refer to ideas of in-depth learning by doing, do-it-yourself and <a href="https://en.wikipedia.org/wiki/Hacker_culture" target="_blank" class="link-external">"playful cleverness"</a> that we can also find present in the term "making". We feel (though others might disagree) that the concepts have enough overlap to discuss them together.
```
This is a brilliant description!

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+### Asking Who Is Excluded
+
+We must consider <a href="WhatIsTheApproach.md#whoIsExcluded">who might be excluded?</a>, and how to redesign our approach to running, promoting and recruiting for our events.
```
We have "we must" here and in the previous sentence too. This sounds very serious. :)

Perhaps, instead, we can say something about how people are sometimes inadvertently included from these kinds of events, and that there are strategies to help minimize this.

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+We must consider <a href="WhatIsTheApproach.md#whoIsExcluded">who might be excluded?</a>, and how to redesign our approach to running, promoting and recruiting for our events.
+
+This is especially critical for hackathons trying to tackle social challenges faced by identifiable groups such as people with disabilities, under the principle of <a href="https://en.wikipedia.org/wiki/Nothing_About_Us_Without_Us" target="_blank" class="link-external">*Nothing About Us Without Us*</a>. Any extra work involved in broadening the reach of the event beyond the "natural" demographic  ...
```
Should we avoid referring to "identifiable groups" here? I'm thinking of people with invisible disabilities, or people who feel they don't want to be primarily identified as having a disability. For many, of course, it's an important part of personal identity, but I guess the point here isn't that it's about "challenges faced by identifiable groups" so much as that we should focus on topics that are defined by people with disabilities as relevant, important, and engaging.

waharnum replied:
It's a good point - current phrasing is definitely problematic. I've rewritten the sentence a few times trying to get it to sound right and am still not happy with it. Writing thoughts out here (less for feedback then to get them recorded somewhere outside my own head)...

- I think that "nothing about us without us" is an important principle for all kinds of (for lack of better phrasing) technocratic engagement with social problems that hackathons often raise. I'm thinking not just of disabilities, but of poverty, food security, racism, gender discrimination... How do we get across the idea that those most directly affected by social challenges have their own kind of expertise, their own knowledge of great value, that technical solutions to social challenges disconnected from the knowledge and experience of those who experience the social challenges will at best be of limited effectiveness and may even be actively harmful?

- Focusing on topics that are relevant, important and engaging - how do we start to engage the individuals who can share this knowledge with us? Most of us who have the capacity to run or participate in a hackathon already have a considerable level of privilege.

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+When considering the design of a hackathon-style event, consider:
+
+* Can you support remote collaboration using audio and video conferencing tools, shared document workspaces, and/or collaborative coding environments?
```
Can you suggest any of these, especially ones that are open source or freely web-based (maybe things like talky.io, etc.)

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+When considering the design of a hackathon-style event, consider:
+
+* Can you support remote collaboration using audio and video conferencing tools, shared document workspaces, and/or collaborative coding environments?
+* Are there alternate styles of event that would also be appropriate, such as the distributed approach of the <a href="http://globalgamejam.org/" target="_blank" class="link-external">Global Game Jam</a>?
```
Also, is there something about be distributed that actually might make an event stronger? Perhaps by fostering the sort of diversity that can emerge when people from every different places and experiences work together on a project?

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+### The Curb Cut Effect and Innovation from the Margins
+
+There are a number of practical reasons to engage with individuals with "extreme" needs that falls outside the "average", along with the moral imperatives of inclusion.
```
Somehow this sentence feels awkward to me. Is this point here that by working with diverse individuals, including people with disabilities, our designs are more innovative, resilient, flexible, and usable for all? i.e. that design for inclusion is a catalyst for innovation and it often causes a "ripple effect" of unanticipated social _and_ technical benefits

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+Secondly, designs intended to serve the needs or goals of individuals on the margin can turn out to be of much broader use. This is sometimes referred to as the *curb-cut effect* after the sidewalk ramps that were originally intended for wheelchair users, but have ended up being of significant benefit to many other users of public spaces.
+
+### Considering the Full Range of Human Diversity
```
I think it might be helpful to move this section further up, since it gets at some of the questions I raised in the previous section.

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+### Considering the Full Range of Human Diversity
+
+Because we define Inclusive Design as design that considers the full range of human diversity with respect to ability, language, culture, gender, age and other forms of human difference, our vision for inclusive hackathons and hacking must encompass more than the (vital!) practice of accomodation for people with disabilities through the design of the physical space, the preparation of alternate f ...
```
Missing "m" in "accommodation."

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+Because hacking and making are strongly associated with STEM (Science, Technology, Engineering and Math) disciplines, it's not possible to discuss this subject without reference to the widespread work that has gone on to investigate and address the issue of demographic diversity in STEM both academically and <a href="https://www.wired.com/brandlab/2015/05/5-numbers-explain-stem-diversity-matters- ...
+
+The STEM acronym itself has come under some criticism for excluding art and design, leading to initiatives such as the <a href="http://stemtosteam.org/" target="_blank" class="link-external">STEM to STEAM Initiative</a>.
```
nice!

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+The STEM acronym itself has come under some criticism for excluding art and design, leading to initiatives such as the <a href="http://stemtosteam.org/" target="_blank" class="link-external">STEM to STEAM Initiative</a>.
+
+We encourage those considering hacking and making in learning contexts to be aware of larger issues of diversity in STEM subjects, workplaces and communities, as these issues can have relevance and knock-on effects to the diversity and inclusion of hackathons and hacking.
```
Is it possible that there are benefits to hacking and making for STEM inclusion? If so, what are they?

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+## Tools
+
+The sections below include examples of tools (software, hardware and design-related) that can be useful in inclusive design hacking. Some are early-stage research projects, while others are mature and in widespread usage in and outside the classroom.
```
How about: "The sections below include examples of software, hardware, and design tools that may be useful for inclusive design hacking."?

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+The sections below include examples of tools (software, hardware and design-related) that can be useful in inclusive design hacking. Some are early-stage research projects, while others are mature and in widespread usage in and outside the classroom.
+
+We have tried to highlight tools that fall somewhere at the nexus of openness, active development, strong communities, and adaptability. Separately or in combination, they might be used to:
```
nice!

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+#### The Nexus
+
+The <a href="https://github.com/GPII/nexus" target="_blank" class="link-external">Nexus</a> is an integration technology based on the <a href="https://github.com/fluid-project/infusion" target="_blank" class="link-external">Infusion</a> Javascript framework that allows <a href="https://wiki.gpii.net/w/Nexus_API" target="_blank" class="link-external">distributed systems to be connected together th ...
```
I would recommend linking to this page instead: https://wiki.gpii.net/w/The_Nexus

colinbdclark replied:
I feel like there's something missing about this description of the Nexus—you mention the technology, but less so what it's _for_. I wonder if, with @simonbates' help (or by borrowing from the link above), we could try to situation how the Nexus is important for making something new possible. Yesterday at the community meeting, Simon said something about how "it lets you connect together systems and components that weren't designed to work together, and that might otherwise be incompatible" or something to that effect, and to me this is a powerful description of why the Nexus is needed for these kinds of projects. Equally, it should be said that the Nexus helps to support the creation of new assistive technologies that can reach deeper into the semantics and state of an application.

waharnum replied:
I've gone with the following additional paragraph:

```
The Nexus allows connection and information-sharing between systems and components that weren't designed to work together by allowing output from one system to be transformed into input other systems can interpret and act upon. New forms of assistive technology can be built by using the Nexus as a central relay that can store, broadcast and transform information about the state of various applications, systems or sensors connected to it.
```

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+The <a href="https://github.com/asterics" target="_blank" class="link-external">Assistive Technology Rapid Integration &amp; Construction Set</a> is an open source construction set for assistive technologies.
+
+AsTeRICS can be used to rapidly prototype new forms of assistive technology using a graphical construction environment that runs on Windows or as a <a href="https://github.com/asterics/WebACS" target="_blank" class="link-external">web-based application.</a>
```
An example might be cool here. I know the Austrian teams have run several hacking-type events with AsTeRICs—I wonder if we can include that somehow?

waharnum replied:
This one seems good, and it has a video: http://www.ki-i.at/helicopter/

waharnum replied:
http://www.asterics-academy.net/ also has a number of examples.

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+#### The Developer Space
+
+The three examples above are only a sample of what is available; more can be found by exploring the <a href="http://dspace.teco.edu/" target="_blank" class="link-external">Developer Space</a>, especially the lists of <a href="http://dspace.teco.edu/en/categories/parts-tools" target="_blank" class="link-external">Parts &amp; Tools</a>.
```
Can we describe what the Developer Space is and what it's goals are? I think it is relevant to this article to describe that there is this clearinghouse site that aims to be a place to go to learn about and find tools that support accessible development.

waharnum replied:
OK - I'll add some content on this. 

While we're at it, can we confirm what URL should be used at this time for the Developer Space? 

Currently, it's http://dspace.teco.edu/ - I've also seen http://staging.developerspace.gpii.net/ used (it has a DOWN FOR MAINTENANCE message currently), and http://developerspace.gpii.net/ appears to be active, but has only the default nginx page.

colinbdclark replied:
@michelled pointed me to this URL, which she thinks is the latest version: http://testing.developerspace.gpii.net/

waharnum replied:
@colinbdclark @michelled is it intentional behaviour for http://testing.developerspace.gpii.net/ that I can access no information without creating an account and logging in?

colinbdclark commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,205 @@
+
+#### Specialized Programming Environments
+
+There is a wide range of software built with the intention of supporting more specific programming tasks than "general purpose" programming languages, or providing alternatives to the more familiar text-centric approaches to programming languages.
```
Is there something to be said about how some "specific purpose" programming environments are designed to empower people who would otherwise not identify as programmers or do programming? (e.g. Scratch -> children, Processing -> artists, Max/MSP -> musicians, etc.)

colinbdclark replied:
It looks like you say this later, but it might be worth highlighting this in your summary here.

waharnum commented:
@colinbdclark I think I've now addressed all your feedback (and the entry is much better for it!) - I believe this should now be mergeable so it will appear on the public site.

I would anticipate doing more work on this in the future, especially as we learn from the various ongoing and upcoming projects like the Social Justice Repair Kit and Big Idea.

colinbdclark commented:
Thanks @waharnum! I'll review it today and then we should be good to go. Thanks so much for your thoughtful work on this.

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+category: Methods
+---
+
+Hackathons, makeathons and similar community-oriented participatory technology events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world. Ideas of "hacking" and "making" have also been popularized as approaches to creatively and collaboratively solve problems in society, business and government.
```
in 'the' broader world

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+
+### Hackathon
+
+We use the term "hackathon" expansively in this entry to refer to events that emphasize intensive short-term collaboration between small groups. The groups typically brainstorm, design and prototype around social, cultural, or political issues that are relevant to them (for example <a href="http://arthackathon.co.uk/" target="_blank" class="link-external">art</a> or <a href="http://civictech.ca/2 ...
```
Maybe the plural 'results'? Also, the double 'to' is still present.

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+While these events are often focused on producing or proposing new software or hardware as a means of addressing a social problem or creating a new product, this is not stricly necessary; hackathons have been held to produce
+<a href="https://www.policyhack.com.au/" target="_blank" class="link-external">government policy</a>, or ideas to address <a href="http://foodinnovationprogram.org/hacking-hunger/" target="_blank" class="link-external">food security</a>.
+
+In learning settings, hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution. These events can be highly engaging to students by allowing them to design and pursue challenges of thei ...
```
'interest' instead of 'interesting'

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+
+## Inclusively Designing Hackathons and Hacking
+
+We believe that that the connections and ideas that get made between people attending hackathons and similar events are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
```
Double 'that'

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+
+### The Curb Cut Effect and Innovation from the Margins
+
+There are no "average" users in inclusive design's philosophy. Per the first dimension of inclusive design, <a href="http://idrc.ocadu.ca/index.php/resources/idrc-online/library-of-papers/443-whatisinclusivedesign" target="_blank" class="link-external">"As individuals spread out from the hypothetical average, the needs of individuals that are outliers, or at the margins, become ever more diverse. ...
```
Missing 'm' in accommodate. Maybe change the second occurrence to 'address'.

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+
+#### Inclusive Design Guide
+
+The <a href="https://guide.inclusivedesign.ca/" target="_blank" class="link-external">Inclusive Design Guide</a> is an evolving collection of insights, practices, tools and activities that can be applied a wide range of design challenges in the digital, physical and service realm.
```
Missing word:  applied 'to' a wide range

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+
+##### Electronics and Internet of Things Prototyping
+
+Inexpensive systems for prototyping electronics have become widespread in the last decade. They can be programmed to operate lights, sensors, motors and other electronic devices; by connecting them to the internet, they can be used to prototype and <a href="https://en.wikipedia.org/wiki/Internet_of_things" target="_blank" class="link-external">internet of things</a> devices.
```
Extra 'and': they can be used to prototype internet of things devices

michelled commented on src/documents/InclusiveMakingAndHacking.html.md.handlebars:
```diff
@@ -0,0 +1,227 @@
+
+Individuals can have unique needs that are not well-served by existing commercial technology aimed at people with disabilities. It can be a powerful exercise in one-size-fits-one design to work with a person to improve an assistive technology they use, or create something entirely new, as in the work done by the <a href="http://www.tetrasociety.org/" target="_blank" class="link-external">Tetra So ...
+
+* Using <a href="http://makeymakey.com/" target="_blank" class="link-external">to build alternate computer interface devices.</a>
```
Missing word: Using Makey Makey to build

waharnum commented:
@michelled thanks for the review! I've fixed all the issues you highlighted and tried to improve sentences in a few places.

I think this can now be merged; once we confirm the current Developer Space URL, we can update the document. Currently `http://dspace.teco.edu/` seems the most functional of them.
